opensubtitles/find.go

```diff
@@ -0,0 +1,122 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+// FindService provides access to the find related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#find
+type FindService service
+
+type FeatureOptions struct {
+	//Feature ID of the feature
+	ID string `url:"id,omitempty"`
+	//IMDB ID of the feature
+	ImdbID string `url:"imdb_id,omitempty"`
+	//TMDB ID of the feature
+	TmdbID string `url:"tmdb_id,omitempty"`
+}
+
+type Feature struct {
+	Data []FeatureData `json:"data"`
+}
+
+type AttributesFeature struct {
+	Title           string          `json:"title"`
+	OriginalTitle   string          `json:"original_title"`
+	Year            string          `json:"year"`
+	SubtitlesCounts SubtitlesCounts `json:"subtitles_counts"`
+	SubtitlesCount  int             `json:"subtitles_count"`
+	SeasonsCount    int             `json:"seasons_count"`
+	ParentTitle     string          `json:"parent_title"`
+	SeasonNumber    int             `json:"season_number"`
+	EpisodeNumber   interface{}     `json:"episode_number"`
+	ImdbID          int             `json:"imdb_id"`
+	TmdbID          int             `json:"tmdb_id"`
+	ParentImdbID    interface{}     `json:"parent_imdb_id"`
+	FeatureID       string          `json:"feature_id"`
+	TitleAka        []string        `json:"title_aka"`
+	FeatureType     string          `json:"feature_type"`
+	URL             string          `json:"url"`
+	ImgURL          string          `json:"img_url"`
+	Seasons         []interface{}   `json:"seasons"`
+}
+type FeatureData struct {
+	ID         string            `json:"id"`
+	Type       string            `json:"type"`
+	Attributes AttributesFeature `json:"attributes"`
+}
+
+//Find details for a movie or tv serie specified by an ID.
```

Comment:
Codacy found an issue: [comment on exported method FindService.Feature should be of the form "Feature ..."](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)